src/monodroid/monodroid.targets

```diff
@@ -125,9 +125,16 @@
     <RunParallelCmds Commands="@(_ConfigureRuntimeCommands)" />
   </Target>
 
+  <Target Name="_FindMonoDroidSources">
+    <ItemGroup>
+      <_MonoDroidSources Include="jni\*.cc;jni\*.h;jni\*.hh;jni\**\*.c;" />
+    </ItemGroup>
+  </Target>
+
   <Target Name="_BuildAndroidRuntimes"
-      Inputs="@(AndroidSupportedTargetJitAbi->'$(IntermediateOutputPath)\%(Identity)-Debug\CMakeCache.txt');@(AndroidSupportedTargetJitAbi->'$(IntermediateOutputPath)\%(Identity)-Release\CMakeCache.txt');jni\*.cc;jni\*.h;jni\*.hh;jni\**\*.c;;..\..\build-tools\scripts\Ndk.targets"
-      Outputs="@(AndroidSupportedTargetJitAbi->'$(OutputPath)\%(Identity)\libmono-android.debug.so');@(AndroidSupportedTargetJitAbi->'$(OutputPath)\%(Identity)\libmono-android-checked+asan.debug.so');@(AndroidSupportedTargetJitAbi->'$(OutputPath)\%(Identity)\libmono-android-checked+ubsan.debug.so');@(AndroidSupportedTargetJitAbi->'$(OutputPath)\%(Identity)\libmono-android.release.so');@(AndroidSupportedTargetJitAbi->'$(OutputPath)\%(Identity)\libmono-android-checked+asan.release.so');@(AndroidSupportedTargetJitAbi->'$(OutputPath)\%(Identity)\libmono-android-checked+ubsan.release.so');@(AndroidSupportedTargetJitAbi->'$(IntermediateOutputPath)\%(Identity)-Debug\libxamarin-app.so');@(AndroidSupportedTargetJitAbi->'$(IntermediateOutputPath)\%(Identity)-Release\libxamarin-app.so')">
```

Comment:
The line width here -- 789 columns! -- makes me think we should pull this out into a new item group, ditto the `Inputs`.

Reply:
This target is using batching at the target level...

I don't think we can actually do this or this batching won't work:

https://github.com/xamarin/xamarin-android/blob/087c7f43a1a60a6c92934d0a57c902e381877df3/src/monodroid/monodroid.targets#L133

If we created `@(_BuildAndroidRuntimesInputs)` and `@(_BuildAndroidRuntimesOutput)`, then there would not be a way for `%(Identity)` to be used.

I don't know how to clean this up without rewriting... I didn't really want to go that far.

Reply:
Nevermind, I think I just got it to work.

Looking at the target, I thought it was using [partial builds](https://github.com/xamarin/xamarin-android/blob/master/Documentation/guides/MSBuildBestPractices.md#building-partially), but it isn't. These targets don't really need to build partially, as if you change a `.cc` file all of the architectures have to rebuild anyway.